### ISSUE TITLE
After installing green on Windows from pip it fails to run

### ISSUE THREAD

**Issue by mairsbw** [CleanCut/green#127]
I installed `green` using pip with administrator permissions on my Windows 10 x64 machine where Python3.5 is install into Program Files. After installing if I run `green green.examples` I see only `failed to create process` and the program returns. But if I run it like `python -m green green.examples`, then it works correctly. It looks like that program is an executable, so it shouldn't be a Python pathing issue. Any ideas what's going on here?

**Comment by CleanCut** [comment]
I don't use Windows!  So...no, I really have no idea.

@MinchinWeb @jayvdb Do you guys have any ideas on this?

**Comment by MinchinWeb** [comment]
I can confirm the issue. I tried running `green --version` on a fresh system going back to `green` 1.8 and the issue ('failed to create process`) came up each time. My guess is there is some dependency that is not being installed...

**Comment by MinchinWeb** [comment]
I suspect it may be this issue with using pip to install --> https://github.com/pypa/pip/issues/2783

The issue is that the green launcher is being install into a directory with a space in the path. To workaround, I would suggest you download the code, and then install manually using `python setup.py install`. Let me know if that works.

**Comment by jayvdb** [comment]
Maybe this is the underlying problem, and is now fixed?

https://github.com/pypa/setuptools/issues/398#issuecomment-225444249

**Comment by CleanCut** [comment]
Unless someone knows otherwise, I will consider this as a bug in setuptools on Windows that has been fixed in some recent setuptools release.

**Comment by jayvdb** [comment]
@mairsbw , @MinchinWeb , can you check whether you have installed python in a directory with spaces in the name?
I will have access to a decent Windows maxhine tomorrow.

But we should be able to reproduce this on Appveyor.

**Comment by MinchinWeb** [comment]
@jayvdb The default installation directory for Python 3.5 on Windows is now the *Program Files* directory, so my Python 3.5 is installed to `C:\Program Files\Python 3.5\python.EXE`.

My *green* wasn't working, so I uninstalled it with *pip* and then reinstalled it (just now), and *green* seems to be working now. So hopefully this is now fixed in *setuptools*, and remains that waypip . I'm currently running Windows 10x64, Python 3.5.2, pip 8.1.2, setuptools 24.0.3, and green 2.5.1.
